docs/UserGuide.md

```diff
@@ -167,14 +167,14 @@ Format: ‘find n/NAME p/PHONE_NUMBER r/ROOM_NUMBER t/Tags’’
 * Persons matching at least one keyword will be returned (i.e. `OR` search).
```

Comment:
The `OR` search is only implemented for names and not for tags. Should this be written into the UG, or fixed within the tags?